src/call/mod.rs

```diff
@@ -616,59 +616,78 @@ impl WriteFlags {
     }
 
     /// Get whether buffer hint is enabled.
-    pub fn get_buffer_hint(self) -> bool {
+    pub fn get_buffer_hint(&self) -> bool {
```

Comment:
Why change?

Reply:
No you don't. WriteFlags is copy and fits in register. It doesn't need to be accessed by pointer indirectly.